src/Common/src/System/IO/PathInternal.Unix.cs

```diff
@@ -16,6 +16,11 @@ internal static partial class PathInternal
 
         internal static readonly int MaxComponentLength = Interop.Sys.MaxName;
 
+        internal const string ParentDirectoryPrefix = @"../";
+
+        internal static readonly StringComparison DefaultPathComparison = StringComparison.Ordinal;
```

Comment:
As I noted yesterday, we also already have internal support for this, which we should just reuse:
https://github.com/dotnet/corefx/blob/master/src/Common/src/System/IO/PathInternal.CaseSensitivity.cs